README.md

```diff
@@ -1,40 +1,6 @@
 # TutorialToken Truffle Box
 
-This box has all you need to get started with our [Open Zeppelin (TutorialToken) tutorial](http://truffleframework.com/tutorials/robust-smart-contracts-with-openzeppelin).
+## This box is deprecated. It is likely some features in this box will not work as expected.
 
-## Installation
+Consider following [this tutorial](https://forum.openzeppelin.com/t/create-an-erc20-using-truffle-without-writing-solidity/2713) to create an ERC20 with truffle.
 
-1. Install Truffle globally.
```

Comment:
I wouldn't delete the old box readme, in case for some reason someone wants to see it. Just adding the deprecation message (maybe make it bold to stand out?) is probably fine!